### DISCUSSION TITLE
Does QField community account only allow 100MB of storage?

### DISCUSSION THREAD

**Discussion by cez2020** [opengisch/QField#3992] [Q&A]
My organization was counting on the 100GB QField was offering. Our projects include large .mbtiles files with multi zoom-level satellite images covering entire administrative districts--we need this for offline data collection in remote parts of Africa. But now, after the release, our projects are basically obsolete as we are limited to only 100MB. We are willing to upgrade, though. Any thoughts or ideas? I hope the QField admins read this and offer a solution.

Thanks!

**Comment by OBVYamaska** [comment]
I just stumbled across the same issue; we heavily tested QFieldCloud this last year and completely switched our process to include the tool. But today I wanted to try something simple, and was blocked every step of the way because of the storage limit and the impossibility to share private projects with collaborators. When I tried to clean some older project to make room, I couldn't even see the size and files of my larger projects, so I had to delete some blindly... not cool.

I must say, even though it makes sense that some things change with the end of the BETA, I'm disappointed that we were not informed of these major changes early on. Our organization convinced about a hundred users to switch to QFieldCloud from ESRI, but with many of them caught off guard with this, we are hearing many speaking about switching back...
Similarly, the solution to create paid accounts for everyone of these partners significantly change what we built, as most of them are small NGO for who the pricing might seem to high for the kind of usage they are doing...

We were hoping to deploy some crowdsourcing solutions with this platform, but I fear that this new approach from QFieldCloud makes it impossible (security reasons prevents us to post some projects as "community projects").

I hope we can find some kind of solution. We are willing to pay, but the options provided clearly don't answer our needs...

**Reply by cez2020** [reply]
I agree! I think some consideration should be given to those of us who tested the app during the Beta phase. I work for a large organization in international development and we have several ongoing data collection projects in developing countries around the world. I discovered QField several months ago and have showcased my QField projects to many teams in the company. Now with 100MB, QField is basically useless. I wouldn't mind paying for a Pro or an organization account for the duration of the data collection phase, but given the amount of storage my project requires, we're considering switching to other apps. I'm highly disappointed :/

**Reply by OBVYamaska** [reply]
I received an notification with this message:
`Hi,
The end of the beta was communicated and an offer for the beta tester was done.
You should have received emails about it.
I'm sorry to read that you are disappointed, but it was also asked to not work in prod during beta test.
I hope you will find a solution with our product.
Best regards.
`

Just to clarify: the end of the Beta was clearly announced, I'm not disputing that.
But the implications were all but clear. I never understood there would be major downgrading on the community accounts. I believe a list or a better description of the modifications should have been done. I didn't use the "bonus" you gave to the beta tester because I never thought I needed it...
I still believe your product is really good, but this lack of information took many of us by surprise, to a point I doubt I'll be able to keep all my partners from switching to other apps. This is what most people fear from opensource projects, and what I fought to convince them would not happen...

Please, at least don't do that again? If such changes have to happen, clearly announce them...

**Comment by mbernasocchi** [comment]
TL;DR
QFieldCloud free community plan has 100MB storage, QFieldCloud Organisation and personal PRO plan can have as much storage as you need

Elaborating on my previous colleague's answer: ~~"Hi, The end of the beta was communicated and an offer for the beta tester was done. You should have received emails about it. I'm sorry to read that you are disappointed, but it was also asked to not work in prod during beta test. I hope you will find a solution with our product. Best regards."~~

Dear @cez2020 and @OBVYamaska,
We're really happy that you enjoy using QField and sorry that you find the QFieldCloud offering not up to your desires. I'll talk about your concerns individually below.
But first, let me give you some insights into what you are getting for free (and completely open source):
- the leading open-source fieldwork app with no strings attached. You can use it with as many users as you want for whatever you want, how much you want, with no obligation to use the QField.cloud service.
- a synchronisation service that you can deploy on your servers and do all you want with it without any limitations.
- a synchronisation plugin to help you package your projects for use with your/our cloud or directly with any other synchronisation tool.

We (OPENGIS.ch) invested 11 years of coding into QField and almost four years in QFieldCloud. And we're not even mentioning all the work that went into QGIS thanks to those projects; all are available to you for free to do all the good you can with it.

If this is still insufficient, we offer a free service for users without strings attached. Yes, it is limited to 100MB of storage, but that allows a user to take care of things she/he cares about. 

If you are doing professional work with these tools, that is great; you have a fantastic toolset to work with. And we would like that you can continue doing so also in future. That is why we also offer a paid service that can help us finance all the other tools you are getting for free. We, like everyone, have to pay bills and servers and developers. 

Now as promised above, let me answer your questions:

> impossibility to share private projects with collaborators

Indeed this has been the case since when we opened the QFIeldCloud BETA; we announced that to share private projects with collaborators, you would have to use an organisation subscription. Meanwhile, we also added the possibility to share private projects across individual PRO accounts.


> I couldn't even see the size and files of my larger projects, so I had to delete some blindly... not cool

That is very good feedback, thanks; in fact, you can see how much space your project is using when you go into the [file tab](https://app.qfield.cloud/a/mbernasocchi/public_test/files) of the project
![image](https://user-images.githubusercontent.com/233663/222743725-f511e4e3-7159-409a-8d7d-e22b0b38a7d2.png)
 but it would definitely be much better to see it on the main project list page to have an overview.

>  I'm disappointed that we were not informed of these major changes early on

We always communicated that a user would enjoy all functions of the organisation plan with no limitations during beta. The Terms Of Service page that we, on purpose, kept super short during BETA explained this very clearly. Furthermore, the 
![image](https://user-images.githubusercontent.com/233663/222749100-daac297f-7e90-4bd9-88a6-7c41b2d869b6.png)

For reference, here is the [april Newsletter](https://mailchi.mp/opengis.ch/seamless-fieldwork-with-qfieldcloud-is-around-the-corner#qfieldcloud). The tentative pricing page with all plans details has also been made public for multiple months before the announcement of the BETA end, and we presented the plans at multiple conferences and presentations.
![image](https://user-images.githubusercontent.com/233663/222772364-6b93a328-22e3-4c0e-bd67-d54b903a82c8.png)

> My organization was counting on the 100GB QField was offering

Please note that the 100MB limit is only for the free community user. Your organisation can still get 100GB of storage; and all projects owned by the organisation will count only in the organisation space. We currently limited the storage selection in the UI to 50GB to be conservative, but we can easily add more if your organisation requires it. Please get in touch with sales@qfield.cloud for that.

> We were hoping to deploy some crowdsourcing solutions with this platform, but I fear that this new approach from QFieldCloud makes it impossible (security reasons prevents us to post some projects as "community projects").

There is no new approach in QFieldCloud; you compare an organisation account with a personal community plan.

>We are willing to pay, but the options provided clearly don't answer our needs...

What exactly do you miss in the organisation plan?

I'm really sorry that all this information did not reach you. Since you were long-time BETA users, we can exceptionally apply the SuperEarlyBee offer that we offered all BETA users. For this, you'll have to send your organisation's name and the link to this message to sales@qfield.cloud.

I hope I could address all your issues and clarified things now. If not, do not hesitate to ask for more details.

Cheers Marco

--
OPENGIS.ch CEO
QGIS.org Chair
OSGeo Board member

**Reply by cez2020** [reply]
Hi Marco

Thank you so much for such a well thought out response. I will soon be sending a message to your sales department and hope to get a subscription set up in a timely manner. Also, thank you for all the hard work you put into developing such an amazing tool!

Best

**Reply by OBVYamaska** [reply]
Thank you for you answer.
At this point, I think the best course of action would be to find  a moment to speak directly. I believe that by presenting what we try to achieve, you will probably be able to point us to the right service. 

We totally understand and respect all the time and energy you put in all this. Your team definitely make the world a better place. We just need time and guidance to adjust to the new structure ;)

I sent an email with some of our concerns yesterday, but I will send a new one to try to find a moment to meet.

Thanks again.

P.S: Just to be clear considering the storage problem: I know I can see the size of my files in the file tab. But yesterday, half my project had info displayed in that tab (the smaller ones), and the other half (the heavier projects) only had the "out of storage" message in that place. It was literally written "0 file". When I deleted my biggest project (I had to do it from memory), I went under the 100MB limit and the other projects finally had the files list and size calculation displayed. Just so you know, if other users come to you with this issue. I guess it can only happen for users that had bigger projects in the Beta before the storage limitation was applied...

**Reply by cez2020** [reply]
Hi again

I recently sent out an email to info@opengis.ch but didn't get a response after I requested scheduling a call to discuss details of your service. I will try the sales email address this time and explain all the details of my data collection project; you might be able to provide a solution that better satisfies our data / service needs. Do you think there's someone in QField that'd be willing to have a call with me, Marco? 

Best!

**Reply by mbernasocchi** [reply]
Hi @cez2020, info@opengis.ch is very high-traffic, so it takes longer to follow up there. sales@qfield.cloud is definitely the best.
Please bear with us as we have a lot of interest currently, so our sales team has much to answer.
I'll make sure you'll get feedback early next week

Have a great weekend
Marco

--
OPENGIS.ch CEO
QGIS.org Chair
OSGeo Board member

**Reply by mbernasocchi** [reply]
> P.S: Just to be clear considering the storage problem: I know I can see the size of my files in the file tab. But yesterday, half my project had info displayed in that tab (the smaller ones), and the other half (the heavier projects) only had the "out of storage" message in that place. It was literally written "0 file". When I deleted my biggest project (I had to do it from memory), I went under the 100MB limit and the other projects finally had the files list and size calculation displayed. Just so you know, if other users come to you with this issue. I guess it can only happen for users that had bigger projects in the Beta before the storage limitation was applied...

thanks for that, we'll look into it and see when this happens.
